gems/pending/appliance_console/timezone_configuration.rb

```diff
@@ -0,0 +1,95 @@
+require "appliance_console/logging"
+require 'appliance_console/prompts'
+
+module ApplianceConsole
+  class TimezoneConfiguration
+    # Timezone constants
+    TZ_AREAS         = %w(Africa America Asia Atlantic Australia Canada Europe Indian Pacific US).freeze
+    TZ_AREAS_OPTIONS = ["United States", "Canada", "Africa", "America", "Asia", "Atlantic Ocean", "Australia", "Europe",
+                        "Indian Ocean", "Pacific Ocean", CANCEL].freeze
+    TZ_AREAS_MAP     = Hash.new { |_h, k| k }.merge!(
+      "United States"  => "US",
+      "Atlantic Ocean" => "Atlantic",
+      "Pacific Ocean"  => "Pacific",
+      "Indian Ocean"   => "Indian"
+    ).freeze
+    TZ_AREAS_MAP_REV = Hash.new { |_h, k| k }.merge!(TZ_AREAS_MAP.invert).freeze
+
+    attr_reader   :cur_city, :cur_loc, :timezone, :tzdata
+    attr_accessor :new_city, :new_loc, :tz_area
+
+    include ApplianceConsole::Logging
+
+    def initialize(config = {})
```

Comment:
Why does this need to be a config hash rather than just passing in the current time zone as a string?